pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -10,6 +10,60 @@ import (
 	f "github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
 )
 
+// DictNew hint creates a new dictionary
+//
+// `newDictNewHint` takes no operander as argument
+func newDictNewHint() hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "DictNew",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> if '__dict_manager' not in globals():
+			//>   from starkware.cairo.common.dict import DictManager
+			//>   __dict_manager = DictManager()
+			//>
+			//> memory[ap] = __dict_manager.new_dict(segments, initial_dict)
+			//> del initial_dict
+
+			//> if '__dict_manager' not in globals():
+			//>   from starkware.cairo.common.dict import DictManager
+			//>   __dict_manager = DictManager()
```

Comment:
Could you share with me the source of this code. I want to look at smth

Reply:
https://github.com/starkware-libs/cairo-lang/blob/efa9648f57568aad8f8a13fbf027d2de7c63c2c0/src/starkware/cairo/common/dict.cairo#L9